pkg/capture/capture_manager.go

```diff
@@ -155,13 +182,14 @@ func (cm *Manager) Status(ctx context.Context, ifaces ...string) (statusmap map[
 		rg             RunGroup
 	)
 	for _, iface := range ifaces {
+		iface := iface
 		mc, exists := cm.captures.Get(iface)
 		if !exists {
 			continue
 		}
 		rg.Run(func() {
 
-			runCtx := logging.WithFields(ctx, slog.String("iface", mc.iface))
```

Comment:
This (and the above `iface := iface` shadow) is unnecessary because the `mc, exists := cm.captures.Get(iface)` call should implicitly "localize" the variable neatly without using the slightly awkward `iface := iface` magic...